legion/engine/llri/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@
 
 project(llri LANGUAGES CXX)
 
-file(GLOB_RECURSE source *.hpp *.inl)
+file(GLOB_RECURSE source *.h *.hpp *.inl)
```

Comment:
what does this do, and shouldn't it also be dependent on a CMake variable?

Reply:
the file command here selects all files of a certain type, in this case I added *.h as a valid file type, in addition to *.hpp and *.inl.

The LLRI project itself only needs header files so I figured just adding the file types manually was sufficient. The reason the other two use a variable is because they **might** need to add *.m and *.mm (Objective C/C++, *.cpp equivalent) files, so those are optionally enabled if the project is built on apple machines